docs/piece-pages.md

```diff
@@ -1,25 +1,20 @@
 ---
-title: "Piece pages"
+title: "Piece Pages"
 ---
 
-# Piece pages in A3
+# Piece Pages
 
-:::  tip Note:
-**"What's a piece page?"**
-<!-- I realize that for alpha 1 our audience is more familiar with Apostrophe, but this is a topic that comes up even among people who know a lot about Apostrophe, so I'd like this note to stay in. Let's work on making it as clear as possible. Tom -->
-
-The most intuitive example is a blog: if the individual [piece](pieces.md) is one blog post, then the piece page is the blog's home page, where all of the posts can be discovered, paginated, filtered and explored. In addition,  "piece pages" are responsible for serving the individual webpage for each piece.
+The most intuitive example of a piece page is an index of products: if the individual [piece](pieces.md) is one product, then the piece page is the product home page, where all of the posts can be discovered, paginated, filtered and explored. In addition, piece pages are responsible for serving the individual page for each piece.
```

Comment:
"product landing page" or "product listing page" rather than "product home page"

Reply:
Yes. It actually got less clear here. Plural became singular, confusing it with the product itself. Even "product landing page" can be for a single product. "Product listing" is better, but everybody knows what a blog is in relation to an individual blog post, which is what we're trying to explain... so why not use it?

Reply:
listing

Reply:
You agreed to listing but it's not changed. I'll change it. ⏩